src/views/footers/local.blade.php

```diff
@@ -0,0 +1,77 @@
+<footer class="py-10 bg-gray-800">
+    <div class="px-4 text-white lg:px-16">
+        <div class="-mx-2 flex flex-wrap justify-between space-y-8 lg:space-y-0">
+            <div class="w-full px-2 lg:w-auto">
+                <a
+                    class="inline-block hover:text-gray-300 focus:outline-none focus:ring focus:ring-blue-400"
+                    href="{{ $logoHref }}"
+                >
+                    {{ $logo ?? 'No logo' }}
+                </a>
+
+                @isset($address)
+                    <address class="mt-4 text-sm not-italic">
+                        {{ $address }}<br>
```

Comment:
I don't think we need this `<br>` here do we?

Reply:
HAHA. What happened there? 😅

Will take it out. It came along with the docs example.

![image](https://user-images.githubusercontent.com/5126648/108633542-c5323e80-7442-11eb-8f94-37571933cc4f.png)